keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -438,19 +450,20 @@ Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
 checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
+By checking the `kube_statefulset_status_replicas_available` metric.
 
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!--
 Pick one more of these and delete the rest.
 -->
 
-- [ ] Metrics
-  - Metric name:
+- [x] Metrics
+  - Metric name: `kube_statefulset_status_replicas_available`
   - [Optional] Aggregation method:
-  - Components exposing the metric:
-- [ ] Other (treat as last resort)
-  - Details:
+  - Components exposing the metric: StatefulSet controller via kube_state_metrics
+
+The `kube_statefulset_status_replicas_available` gives the number of replicas available.
```

Comment:
Can you give a little more detail on the metric labels? Will a timeseries exist for every workload that this is enabled for?

Reply:
Yes. We use labels to identify the individual workload and timeseries would exist for it.